projects/plugins/jetpack/sal/class.json-api-date.php

```diff
@@ -47,7 +47,7 @@ public static function format_date( $date_gmt, $date = null ) {
 						$offset = $timezone->getOffset( $date_time );
 					}
 				} else {
-					$offset = 3600 * get_option( 'gmt_offset' );
+					$offset = 3600 * (int) get_option( 'gmt_offset' );
```

Comment:
I wonder if this one might be a float.

I know there are some timezones that are a non-integer offset from UTC, but I don't know whether this 'gmt_offset' option handles that or not.

Reply:
Good thought; reverted in a159a56dff.

Seems the same thing happened in WP core:
WordPress/WordPress@6e83ac559549a3b1a391286ef57e8680f97d498d
WordPress/WordPress@b1d826fb6e0bd5f8ffcad511658e8de8a60e6a42